src/test/resources/bom-1.2-vulnerability.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This is not a 1.2 schema, just a copy of 1.1 with a vuln, so it'd make sense to actually make it a 1.2 schema and bring in more bits, merge it with having a vuln, etc...